src/js/apps/forms/form/form-history_app.js

```diff
@@ -0,0 +1,56 @@
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import { LayoutView, HistoryBarView, FormView } from 'js/views/forms/form/form-history_views';
+
+export default App.extend({
+  viewEvents: {
+    'change:response': 'onChangeResponse',
+  },
+  stateEvents: {
+    'change:response': 'showIframeView',
+  },
+  onBeforeStart({ response }) {
+    this.showView(new LayoutView());
+    this.getRegion('iframe').startPreloader();
+
+    this.setState('response', response);
```

Comment:
I think we should probably not worry about passing the response in here.  Just set it based on `action.getFromResponses().first()` in the `onStart` as that will ensure the latest response it what is shown.

Reply:
Do you think we should be passing the selected response to the `showIframeView` method then? The reason `response` is set on the state here is because we pass `this.getState('response')`. I can move it to onStart, though.